### PR TITLE
Shared and Auth modules angular

### PR DESCRIPTION
**Pull request by profileEnumerable**

### CODE REVIEW

**Review by antonAce — CHANGES_REQUESTED** [review]
Thank you for this major clean-up! There are some extra suggestions

File: Src/Campus.Master.API/UI/src/app/auth/sing-in/sing-in.component.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <div class="wrap-center">
     <div class="sign-in">
       <div class="sign-in__img">
-        <img src="/assets/img/sign-in/signIn.png" alt="photo"/>
+        <img src="../../../assets/img/sign-in/signIn.png" alt="photo"/>
```
[the remark this answers is not in the document]
Comment (profileEnumerable) [reply Src/Campus.Master.API/UI/src/app/auth/sing-in/sing-in.component.html:5]: reduced

File: Src/Campus.Master.API/UI/src/app/auth/registration-page/registration-page.component.html [hunk 2]
```diff
@@ -2,7 +2,7 @@
   <div class="wrap-center">
     <div class="registration">
       <div class="registration__img">
-        <img src="/assets/img/registration/reg.png" alt="photo">
+        <img src="../../../assets/img/registration/reg.png" alt="photo">
```
[the remark this answers is not in the document]
Comment (profileEnumerable) [reply Src/Campus.Master.API/UI/src/app/auth/registration-page/registration-page.component.html:5]: reduced

File: Src/Campus.Master.API/UI/src/app/auth/shared/services/sign-in.service.ts [hunk 3]
```diff
@@ -1,8 +1,8 @@
 import { Injectable } from '@angular/core';
 import { environment } from 'src/environments/environment';
 import { Observable, Subject, throwError } from 'rxjs';
-import { User } from '../models/user/user';
-import { StateTransfer } from '../models/state-transfer';
+import { User } from '../models/user';
+import { StateTransfer } from '../../../models/state-transfer';
```
[the remark this answers is not in the document]
Comment (profileEnumerable) [reply Src/Campus.Master.API/UI/src/app/auth/shared/services/sign-in.service.ts]: reduced

File: Src/Campus.Master.API/UI/src/app/data/dao/impl/CategoryDAOArray.ts [hunk 4]
```diff
@@ -1,5 +1,5 @@
 import { CategoryDAO } from '../interface/CategoryDAO';
-import { Project } from '../../../shared/models/task-list/project';
+import { Project } from '../../../models/task-list/project';
```
[the remark this answers is not in the document]
Comment (profileEnumerable) [reply Src/Campus.Master.API/UI/src/app/data/dao/impl/CategoryDAOArray.ts:2]: fixed

File: Src/Campus.Master.API/UI/src/app/data/dao/impl/TaskDAOArray.ts [hunk 5]
```diff
@@ -1,10 +1,10 @@
-import { Project } from '../../../shared/models/task-list/project';
+import { Project } from '../../../models/task-list/project';
 import { Observable, of } from 'rxjs';
 import { TaskDAO } from '../interface/TaskDAO';
-import { Task } from 'src/app/shared/models/task-list/task';
+import { Task } from 'src/app/models/task-list/task';
```
[the remark this answers is not in the document]
Comment (profileEnumerable) [reply Src/Campus.Master.API/UI/src/app/data/dao/impl/TaskDAOArray.ts]: fixed

File: Src/Campus.Master.API/UI/src/app/auth/sing-in/sing-in.component.sass [hunk 6]
```diff
@@ -1,4 +1,4 @@
-@import "src/styles/styles"
+@import "../../../styles/styles"
```
[the remark this answers is not in the document]
Comment (profileEnumerable) [reply Src/Campus.Master.API/UI/src/app/auth/sing-in/sing-in.component.sass]: fixed

File: Src/Campus.Master.API/UI/src/app/dialog/edit-task-dialog/edit-task-dialog.component.ts [hunk 7]
```diff
@@ -1,11 +1,11 @@
 import { Component, Inject, OnInit } from '@angular/core';
 import { MAT_DIALOG_DATA, MatDialog, MatDialogRef } from '@angular/material/dialog';
-import { DataHandlerService } from '../../shared/services/data-handler.service';
-import { Task } from '../../shared/models/task-list/task';
-import { Project } from '../../shared/models/task-list/project';
-import { Priority } from '../../shared/models/task-list/priority';
+import { DataHandlerService } from '../../services/data-handler.service';
```
[the remark this answers is not in the document]
Comment (profileEnumerable) [reply Src/Campus.Master.API/UI/src/app/dialog/edit-task-dialog/edit-task-dialog.component.ts:3]: fixed

File: Src/Campus.Master.API/UI/src/app/auth/shared/services/registration.service.ts [hunk 8]
```diff
@@ -1,9 +1,9 @@
 import { Injectable } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
 import { Observable } from 'rxjs';
-import { environment } from 'src/environments/environment';
-import { RegisterUser } from '../models/user/register-user';
-import { StateTransfer } from '../models/state-transfer';
+import { environment } from 'environments/environment';
+import { RegisterUser } from '../models/register-user';
+import { StateTransfer } from '../../../models/state-transfer';
```
Comment (antonAce) [inline Src/Campus.Master.API/UI/src/app/auth/shared/services/registration.service.ts:6]: Just add `@sharedModels` for other references

**Review by antonAce — APPROVED** [review]
Thanks for the major clean-up :shipit:

**Pull request merged**